### PR TITLE
fix(annotation): enable modify annotations

### PR DESCRIPTION
**Pull request by ChenCodes**
Changes in this PR:
- remove flag for disabling modify annotations
- update comment UX 
- update tests

### CODE REVIEW

**Comment by ChenCodes** [comment]
Do not merge until we have translations completed.

**Review by ConradJChan — COMMENTED** [review]
LGTM to merge when possible

**Review by mxiao6 — APPROVED** [review]

**Pull request merged**
